cpu/sam0_common/periph/spi.c

```diff
@@ -281,55 +248,47 @@ static void _spi_acquire(spi_t bus, spi_mode_t mode, spi_clk_t clk)
      * to mitigate the rounding error due to integer arithmetic, the
      * equation is modified to
      * BAUD.reg = ((f_ref + f_bus) / (2 * f_bus) - 1) */
-    const uint8_t baud = (gclk_src + clk) / (2 * clk) - 1;
+    uint8_t baud = (gclk_src + clk) / (2 * clk) - 1;
```

Comment:
why do you unconst this (line seems uncanged otherwise)
same for ctrla 

liter (const correctness) might not like that change

Reply:
For consistency with the other variables holding the values to store into the registers.